mico-core/src/main/java/io/github/ust/mico/core/dto/MicoApplicationDTO.java

```diff
@@ -0,0 +1,185 @@
+            @ExtensionProperty(name = "description", value = "Version number of the application.")
+        }
+    )})
+    private String version;
+
+    /**
+     * Human readable description of this application.
+     */
+    @ApiModelProperty(required = true, extensions = {@Extension(
+        name = CustomOpenApiExtentionsPlugin.X_MICO_CUSTOM_EXTENSION,
+        properties = {
+            @ExtensionProperty(name = "title", value = "Description"),
+            @ExtensionProperty(name = "x-order", value = "40"),
+            @ExtensionProperty(name = "description", value = "Human readable description of this application.")
+        }
+    )})
+    private String description;
+
+
+    // ----------------------
+    // -> Optional fields ---
+    // ----------------------
+
+    /**
+     * Human readable contact information for support purposes.
+     */
+    @ApiModelProperty(extensions = {@Extension(
+        name = CustomOpenApiExtentionsPlugin.X_MICO_CUSTOM_EXTENSION,
+        properties = {
+            @ExtensionProperty(name = "title", value = "Contact"),
+            @ExtensionProperty(name = "x-order", value = "110"),
+            @ExtensionProperty(name = "description", value = "Human readable contact information for support purposes.")
+        }
+    )})
+    private String contact;
+
+    /**
+     * Human readable information for the application owner
+     * who is responsible for this application.
+     */
+    @ApiModelProperty(extensions = {@Extension(
+        name = CustomOpenApiExtentionsPlugin.X_MICO_CUSTOM_EXTENSION,
+        properties = {
+            @ExtensionProperty(name = "title", value = "Owner"),
+            @ExtensionProperty(name = "x-order", value = "100"),
+            @ExtensionProperty(name = "description", value = "Human readable information for the application owner, " +
+                "who is responsible for this application.")
+        }
+    )})
+    private String owner;
+    
+    /**
+     * Indicates whether the {@link MicoApplication} is currently deployed.
+     */
+    @ApiModelProperty(extensions = {@Extension(
+            name = CustomOpenApiExtentionsPlugin.X_MICO_CUSTOM_EXTENSION,
+            properties = {
+                @ExtensionProperty(name = "title", value = "Deployment Status"),
+                @ExtensionProperty(name = "x-order", value = "120"),
+                @ExtensionProperty(name = "description", value = "Holds the current deployment status of this application.")
+            }
+        )})
+    private MicoApplicationDeploymentStatus deploymentStatus;
+    
+    
+    /**
+     * Creates a {@code MicoApplicationDTO} based on a
+     * {@link MicoApplication}.
+     * 
+     * @param application the {@link MicoApplication}.
+     * @return a {@link MicoApplicationDTO} with all the values
+     *         of the given {@code MicoApplication}.
+     */
+    public static MicoApplicationDTO valueOf(MicoApplication application) {
```

Comment:
`deploymentStatus` is not set

Reply:
Intentionally, i don't want the functionality to "compute" the deployment status inside the DTO class. I suggest to set the field to `UNKNOWN`by default - what do you think?

Reply:
ok, sounds reasonable.

Reply:
Ok, it's set to `UNKNOWN`, plus I added a note in the comment that it has to be set explicitly.